src/rtl_tcp.c

```diff
@@ -92,11 +94,12 @@ static int verbosity = 0;
 static uint32_t bandwidth = 0;
 
 static int enable_biastee = 0;
-static int global_numq = 0;
+static atomic_int global_numq = 0;
 static struct llist *ll_buffers = 0;
 static int llbuf_num = 500;
 
-static volatile int do_exit = 0;
+atomic_bool do_exit = false;
```

Comment:
Why the static is remove? It looks like the variable is still only used within this file.